src/variables.jl

```diff
@@ -1000,6 +1000,17 @@ function start_value(v::VariableRef)::Union{Nothing,Float64}
     return MOI.get(owner_model(v), MOI.VariablePrimalStart(), v)
 end
 
+"""
+    has_start_value(variable::VariableRef)
+
+Return `true` if the variable has a start value set otherwise return `false`.
+
+See also [`set_start_value`](@ref).
+"""
+has_start_value(v::VariableRef)::Bool = start_value(v) !== nothing
+    return MOI.get(owner_model(v), MOI.VariablePrimalStart(), v) !== nothing
```

Comment:
```suggestion
```

Reply:
You were too quick on the commit! We need to remove these other lines as well

Reply:
aarrrg.. yeah the impatience. I will go to the computer and this properly